### DISCUSSION TITLE
Example for using with Remix?

### DISCUSSION THREAD

**Discussion by edwin177** [themesberg/flowbite#47] [Q&A] [answered]
Has anyone tried using flowbite with remix? I'd love to see an example getting the custom js file to load.

**Comment by zoltanszogyenyi** [comment, marked as answer]
Hey @edwin177,

I've moved this to the discussions area.

We generally have issues running the Flowbite JS on libraries like React or Vue since they use a virtual DOM. We're trying to find a workaround to make them compatible before we launch an official React & Vue library for Flowbite.

The way Flowbite JS works is that there are event listeners created for the data attributes.

Regarding your question, by the way, is that all of the non-interactive elements (like dropdown, modal) should work with Remix as it only requires Tailwind CSS.

**Reply by edwin177** [reply]
Thanks for the reply. I'll just use your components as examples to copy-paste CSS classes from for now.

Once there's a React library, I'll take another look at including it directly.
